captum/insights/features.py

```diff
@@ -60,8 +61,8 @@ def __init__(
                             specified in ``captum/captum/insights/frontend/App.js``.
         """
         self.name = name
-        self.baseline_transforms = baseline_transforms
-        self.input_transforms = input_transforms
+        self.baseline_transforms = format_transforms(baseline_transforms)
+        self.input_transforms = format_transforms(input_transforms)
```

Comment:
Was checking the usage of these transforms and noticed there are some checks in api.py verifying these (baseline_transforms and input_transforms) are not None, those shouldn't be necessary after this change since the format call will only return a List?